deps.edn

```diff
@@ -10,14 +10,14 @@
          cheshire/cheshire               {:mvn/version "5.12.0"}
          instaparse/instaparse           {:mvn/version "1.4.12"}
          metosin/malli                   {:git/url "https://github.com/metosin/malli.git"
-                                          :git/sha "5211aca498ff7b022c36de4ca713990f5e731a9c"}
+                                          :git/sha "2aac49abffb9005188c70ffff825d88efee18069"}
```

Comment:
Just curious: are we not on the main release branch for malli, or are we ahead of the most recent release? Or is there some other reason to use git coords instead of a release?